tests/foreman/cli/test_remoteexecution.py

```diff
@@ -290,16 +307,145 @@ def test_positive_install_multiple_packages_with_a_job_by_ip(
-                'inputs': 'package={} {} {}'.format(*packages),
+                'inputs': f'package={" ".join(packages)}',
                 'search-query': f'name ~ {client.hostname}',
             }
         )
         assert_job_invocation_result(invocation_command['id'], client.hostname)
         result = client.run(f'rpm -q {" ".join(packages)}')
         assert result.status == 0
+        # Update packages
+        pre_versions = result.stdout.splitlines()
+        result = client.run(f'dnf -y downgrade {" ".join(packages)}')
+        assert result.status == 0
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Update Package - Katello Script Default',
+                'inputs': f'package={" ".join(packages)}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        post_versions = client.run(f'rpm -q {" ".join(packages)}').stdout.splitlines()
+        assert set(pre_versions) == set(post_versions)
+        # Remove packages
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Remove Package - Katello Script Default',
+                'inputs': f'package={" ".join(packages)}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        result = client.run(f'rpm -q {" ".join(packages)}')
+        assert result.status == len(packages)
+
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([8])
+    @pytest.mark.skipif(
+        (not settings.robottelo.repos_hosting_url), reason='Missing repos_hosting_url'
+    )
+    def test_positive_install_remove_packagegroup_with_a_job(
+        self, rhel_contenthost, module_org, module_ak_with_cv, target_sat
+    ):
+        """Run job to install and remove several package groups on host
+
+        :id: 5be2a5c0-199a-4655-9784-e95c7ef151f5
+
+        :expectedresults: Verify the package groups were successfully installed
+            and removed on a host
+
+        :parametrized: yes
+        """
+        client = rhel_contenthost
+        groups = ['birds', 'mammals']
+        client.register(
+            module_org,
+            None,
+            module_ak_with_cv.name,
+            target_sat,
+            repo=settings.repos.yum_1.url,
+        )
+        # Install the package groups
+        invocation_command = make_job_invocation(
+            {
+                'job-template': 'Install Group - Katello Script Default',
+                'inputs': f'package={" ".join(groups)}',
+                'search-query': f'name ~ {client.hostname}',
+            }
+        )
+        assert_job_invocation_result(invocation_command['id'], client.hostname)
+        result = client.run('dnf grouplist --installed')
```

Comment:
I know this is set to only run on RHEL8 for now... but how about using variable for a packaging manager name?